src/test/java/org/jenkinsci/remoting/protocol/ProtocolStackImplTest.java

```diff
@@ -750,6 +747,7 @@ public void socketChannelFullProtocolNIO_serverRejects() throws Exception {
         assertThat(se.getCause(), instanceOf(ConnectionRefusalException.class));
     }
 
+    @Disabled("TODO flake: ConnectionRefusal Incorrect acknowledgement received, expected 0x000341436b got 0x0000000000")
```

Comment:
The exception is expected, but sometimes gets thrown too early (in `release`). Not sure what test should be doing.